jinseok/routes/likeRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const likeController = require('../controller/likeController');
+
+const router = express.Router();
+
+router.post('/', likeController.createDeleteLike);
```

Comment:
```suggestion
router.post('', likeController.createDeleteLike);
```
엔드포인트 끝에는 `/`가 들어가지 않는것이 REST한 api 엔드포인트입니다.